README.md

```diff
@@ -1,6 +1,89 @@
-# JanusGraph Python driver
+# JanusGraph-Python
 
-## License
+## Python Client drivers for [JanusGraph](http://janusgraph.org)
+
+JanusGraph-Python is the Python drivers for connecting to JanusGraph. 
+It extends Apache TinkerPop™'s [Gremlin-Python](http://tinkerpop.apache.org/docs/current/reference/#_gremlin_python) 
+as its core dependency with additional support for JanusGraph-specific types and predicates.
+
+[![Codacy Badge](https://api.codacy.com/project/badge/Grade/a643a483556740c1b87ba29e160d37b6)](https://www.codacy.com/app/JanusGraph/janusgraph-python?utm_source=github.com&utm_medium=referral&utm_content=JanusGraph/janusgraph-python&utm_campaign=Badge_Grade)
+
+### Pre-Requisites
```

Comment:
Why is building of the library explained here if there is also a `BUILDING.md`? If you use a `BUILDING.md` to explain how to build the library (which I personally prefer), then please keep the README focused on things users are interested in.

Reply:
Well did you mean Pre-requisite section @FlorianHockmann ? This is specific to versions of Python being supported by this library. This is kinda generic I personally feel because no matter if you are building the library, or installing from PyPi at later stage, you need to make sure the Python version in your system is specified here.

But if you feel the version information should also be moved to BUILDING docs then it should not be issue.

Reply:
This section sounds to me like it's mostly written for contributors who want to build the library. E.g.:

>Make a note of the number, as that will be required during building of Library (on UNIX)
[...]
NOTE: The above checks aren't needed if installing the drivers from PyPi

Just mentioning that the library has been tested with Python versions 3.4, 3.5, and 3.6 should be enough in my opinion here.